docs/rules.md

```diff
@@ -35,6 +35,55 @@ extended configuration file as well, to pass them both to the TypeScript compile
 | <a id="ts_config-src"></a>src |  The tsconfig.json file passed to the TypeScript compiler   | <a href="https://bazel.build/docs/build-ref.html#labels">Label</a> | required |  |
 
 
+<a id="#ts_project_rule"></a>
```

Comment:
Can this go at the bottom after the standard `ts_project`?

Reply:
no, we can't control the order of the stardoc generation AFAIK

Reply:
It's not the order the rules are declared in the file? (which would be a different commit anyway imo)